usama/sprint1/infra/infra_stack.py

```diff
@@ -0,0 +1,35 @@
+# For consistency with other languages, `cdk` is the preferred import name for
+# the CDK's core module.  The following line also imports it as `core` for use
+# with examples from the CDK Developer's Guide, which are in the process of
+# being updated to use `cdk`.  You may delete this import if you don't need it.
+
+from aws_cdk import core as cdk
+from aws_cdk import core
+from aws_cdk import aws_lambda as lambda_
+from aws_cdk import aws_events
+from aws_cdk import aws_events_targets
+
+
+
+class InfraStackUsama(cdk.Stack):
+
+    def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
```

Comment:
You can find both the latency and availability by just function:
- Here you're creating PoolManager and getting HTTP response twice
- It'll save your time and memory as well